platform/metrics/metrics-servlet-filter/pom.xml

```diff
@@ -44,6 +44,12 @@
             <artifactId>slf4j-api</artifactId>
             <version>${org.slf4j.version}</version>
         </dependency>
+        <dependency>
+            <groupId>ddf.platform</groupId>
+            <artifactId>platform-paxweb-jettyconfig</artifactId>
```

Comment:
❓ Is this dependency actually needed here?  It should only depend on the API bundle with the `HttpFilter` interface.

Reply:
You're right. I'd originally made the new interface in the paxweb-jettyconfig bundle and forgot to clean this up.